examples/post-process/bloom.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Unreal Bloom Effect
+ * Implementation for A-Frame
+ * Code modified from Akbartus's UnrealBloomPass.js
+ * https://github.com/akbartus/A-Frame-Component-Postprocessing
+ */
+
+import AFRAME from "aframe";
+import { EffectComposer } from "three/addons/postprocessing/EffectComposer.js"; // This uses the same three instance.
+import { RenderPass } from "three/addons/postprocessing/RenderPass.js"; // This uses the same three instance.
+import { UnrealBloomPass } from "three/addons/postprocessing/UnrealBloomPass.js"; // This uses the same three instance.
+
+/////////////////////////////////////
+// A-FRAME COMPONENT: POST-PROCESSING //
+/////////////////////////////////////
+AFRAME.registerComponent("bloom", {
+  schema: {
+    threshold: {
+      type: "number",
+      default: 0,
+    },
+    strength: {
+      type: "number",
+      default: 0.4,
+    },
+    radius: {
+      type: "number",
+      default: 0,
+    },
+    exposure: {
+      type: "number",
+      default: 1,
+    },
+  },
+  init: function () {
+
+    this.scene = this.el.object3D;
+    this.renderer = this.el.renderer;
+    this.camera = this.el.camera;
+    this.bound = false;
+
+    var that = this;
+
+    that.evaluateEffect();
+    that.bind();
+  },
+  update: function (oldData) {
+    console.log("Post-processing UPDATE effect");
+    this.evaluateEffect();
+  },
+  evaluateEffect: function () {
+    this.scene = this.el.object3D;
+    this.renderer = this.el.renderer;
+    this.camera = this.el.camera;
+    this.composer = new EffectComposer(this.renderer);
```

Comment:
Instead of recreating `EffectComposer`, `RenderPass` and `UnrealBloomPass` each update, they can be created once in `init` and only updated here.